Challenge1/package.json

```diff
@@ -33,7 +33,8 @@
     "format": "prettier --write \"src/**/*.ts\""
   },
   "dependencies": {
-    "forta-agent": "^0.1.48"
+    "forta-agent": "^0.1.48",
+    "forta-agent-tools": "^3.2.12"
```

Comment:
From what I can see, the bot isn't using `forta-agent-tools` in production (i.e. it is only using it in the test file). As such, `forta-agent-tools` should be moved to `"devDependencies"`.